pkg/inventory/inventory-client.go

```diff
@@ -121,7 +121,7 @@ func (cic *ClusterInventoryClient) Merge(localInv InventoryInfo, objs []object.O
 		if err != nil {
 			return pruneIds, err
 		}
-		if object.SetEquals(objs, clusterObjs) {
+		if object.IsSetEqual(objs, clusterObjs) {
```

Comment:
Remove this change. If you want to advocate for a name change, please do it in another PR.